### ISSUE TITLE
Lightbox - Accessibility contrast issue in dialog title

### ISSUE THREAD

**Issue by duboisp** [wet-boew/wet-boew#7731]
The contrast of the title in the dialog do not meet WCAG.

To reproduce:
Start here: http://wet-boew.github.io/v4.0-ci/demos/lightbox/lightbox-en.html
Click on the second item under "Singles items" heading named "AJAX - Example 1"
The contrast ratio of the dialog title is 1.55 where the background color is #2e5274 and the color is #333.

I propose to set a default color like this:
```
.modal-title {
	color: #fff;
}
```
in https://github.com/wet-boew/wet-boew/blob/master/src/plugins/lightbox/_base.scss
or in https://github.com/wet-boew/wet-boew/blob/master/src/plugins/overlay/_base.scss

@cfarquharson what do you think of using the color white on a background #2e5274?

Cheers

**Comment by duboisp** [comment]
I found this issue is a regression introduced by this commit: https://github.com/wet-boew/wet-boew/commit/f262c57e86ebf4b4b8c14a39116f6b8616568bc2 merged by PR #7668

**Comment by masterbee** [comment]
@duboisp - good catch this is actually a bit more wide spread that just lightbox, most of the overlays by default are effected (actually anything with the modal design pattern). I am ok with leveraging white for this default choice, but lets see what @cfarquharson has to say.

/cc @cfarquharson

**Comment by cfarquharson** [comment]
Agreed!  Definitely fails accessibility colour contrast requirements in its current state.  Text should be white.
@duboisp @masterbee 

C

**Comment by masterbee** [comment]
@cfarquharson - thanks for getting back us. tag we are it.
